handler_templates.ts

```diff
@@ -332,6 +332,58 @@ export function transformBody(body: any, schema: Schema | undefined): any {
   return body;
 }
 
+function unmapKeys(obj: {[key: string]: any}, schema?: Schema): object {
+  if (!(schema && isObject(schema))) {
+    return obj;
+  }
+
+  const {properties} = schema;
+
+  // Look at the properties of the schema and invert any keys if present.
+  const remappedKeys: Map<string, string> = new Map();
+  for (const key in properties) {
```

Comment:
Nit, does `Object.keys(properties)` already restrict to only own-properties? If used throughout, could eliminate the need for hasOwnProperty() checks.

Reply:
Yeah, it restricts to own enumerable properties.  Probably not a bad idea, but I'd prefer to be consistent with code in the existing `mapKeys`.  Perhaps we come back and fix them both up.